django_db_logger/db_log_handler.py

```diff
@@ -46,3 +49,30 @@ def format(self, record):
             return fmt.formatMessage(record)
         else:
             return fmt.format(record)
+
+    def add_details(self, record, log_fields):
```

Comment:
I would usggest to remove `DJANGO_DB_LOGGER_LOG_REQUEST_DETAILS` and also leave the method empty:

```python
def add_details(self, record, log_fields):
    pass
```

The exmaple in the readme is great, but should be only related to the `add_details` method.

`extract_request_details` also goes out of the scope of adding the `details` JSON field to the module

Reply:
Removed ``DJANGO_DB_LOGGER_LOG_REQUEST_DETAILS`` and ``extract_request_details`` completely, added an example for ``add_details`` to the README.